receiver/adapter/accumulator/accumulator.go

```diff
@@ -188,9 +185,13 @@ func (o *otelAccumulator) modifyMetricAndConvertToOtelValue(m telegraf.Metric) (
 	// Otel only supports numeric data. Therefore, filter unsupported data type and convert metrics value to corresponding value before
 	// converting the data model
 	// https://github.com/open-telemetry/opentelemetry-collector/blob/bdc3e22d28006b6c9496568bd8d8bcf0aa1e4950/pdata/pmetric/metrics.go#L106-L113
+	var errs error
 	for field, value := range mMetric.Fields() {
 		// Convert all int,uint to int64 and float to float64 and bool to int.
-		otelValue := util.ToOtelValue(value)
+		otelValue, err := util.ToOtelValue(value)
+		if err != nil {
+			errs = multierr.Append(errs, fmt.Errorf("field (%s): %w", field, err))
```

Comment:
It looks like if at least 1 field/value is successfully converted then no error is returned.
So errors are ignored unless every single field/value fails to get converted.
Does this match previous behavior?

Reply:
Yes, we only return the error if all the fields were dropped.